autoscale.py

```diff
@@ -1,46 +1,85 @@
-import psycopg2
+import asyncio
+import logging
 import os
 from collections import defaultdict
+from types import NoneType
+from typing import Literal as StringLiteral
+from typing import TypedDict, cast
+
+import psycopg
 from dotenv import load_dotenv
+from psycopg.sql import SQL, Identifier, Literal
+
+logging.basicConfig(level=logging.DEBUG)
+
+LOGGER = logging.getLogger("")
+
+formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+LOGGER.handlers = []
+LOGGER.addHandler(handler)
 
 # Load environment variables from .env file
 load_dotenv()
 
 # Configuration variables
-DB_HOST = os.getenv('DB_HOST', 'localhost')
-DB_PORT = os.getenv('DB_PORT', '6875')
-DB_USER = os.getenv('DB_USER', 'materialize')
-DB_PASSWORD = os.getenv('DB_PASSWORD', 'password')
-DB_NAME = os.getenv('DB_NAME', 'materialize')
+DB_HOST = os.getenv("DB_HOST", "localhost")
+DB_PORT = os.getenv("DB_PORT", "6875")
+DB_USER = os.getenv("DB_USER", "materialize")
+DB_PASSWORD = os.getenv("DB_PASSWORD", "password")
+DB_NAME = os.getenv("DB_NAME", "materialize")
 
-THRESHOLD_UPPER = int(os.getenv('THRESHOLD_UPPER', '180'))  # Scale up if utilization exceeds this percent
-THRESHOLD_LOWER = int(os.getenv('THRESHOLD_LOWER', '40'))   # Scale down if utilization falls below this percent
+THRESHOLD_UPPER = int(
+    os.getenv("THRESHOLD_UPPER", "20")
+)  # Scale up if utilization exceeds this percent
+THRESHOLD_LOWER = int(
+    os.getenv("THRESHOLD_LOWER", "120")
```

Comment:
curious the reasoning behind the default threshold changes and why upper is less than lower

Reply:
Good eye.. I just missed rolling this back after some testing